addons/payment_stripe_sca/models/payment.py

```diff
@@ -58,7 +58,7 @@ def _stripe_request(self, url, data=False, method="POST"):
         # cfr https://stripe.com/docs/error-codes
         # these can be made customer-facing, as they usually indicate a problem with the payment
         # (e.g. insufficient funds, expired card, etc.)
```

Comment:
Modify this comment to detail the thinking going on under here:
```suggestion
        # these can be made customer-facing, as they usually indicate a problem with the payment
        # (e.g. insufficient funds, expired card, etc.)
        # if the context key `stripe_manual_payment` is then these error will be raised as ValidationError,
        # otherwise they will be silenced and the will be returned no matter the status.
        # This key should typically by set for payments the present and unset for automated payments
        # (e.g. through crons)
```